docs/source/memory.rst

```diff
@@ -36,9 +36,9 @@ The following functions produce Futures
    Client.persist
    Client.scatter
 
-Submit and map handle raw Python functions.  Compute and persist handle Dask
-collections like arrays, bags, delayed values, and dataframes.  Scatter sends
-data directly from the local process.
+The ``submit`` and ``map`` methods handle raw Python functions.  Compute and
```

Comment:
Sorry, I forgot to mention the next sentences were also affected :-) ("Compute and persist", "Scatter")